dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -553,21 +556,8 @@ pub fn invoke_resolve(resource: &ResourceManifest, cwd: &str, input: &str) -> Re
     Ok(result)
 }
 
-/// Invoke a command and return the exit code, stdout, and stderr.
-///
-/// # Arguments
-///
-/// * `executable` - The command to execute
-/// * `args` - Optional arguments to pass to the command
-/// * `input` - Optional input to pass to the command
-/// * `cwd` - Optional working directory to execute the command in
-///
-/// # Errors
-///
-/// Error is returned if the command fails to execute or stdin/stdout/stderr cannot be opened.
-#[allow(clippy::implicit_hasher)]
-pub fn invoke_command(executable: &str, args: Option<Vec<String>>, input: Option<&str>, cwd: Option<&str>, env: Option<HashMap<String, String>>, exit_codes: &Option<HashMap<i32, String>>) -> Result<(i32, String, String), DscError> {
-    debug!("Invoking command '{}' with args {:?}", executable, args);
+async fn run_process_async(executable: &str, args: Option<Vec<String>>, input: Option<&str>, cwd: Option<&str>, env: Option<HashMap<String, String>>, exit_codes: &Option<HashMap<i32, String>>) -> Result<(i32, String, String), DscError> {
```

Comment:
even if this is no longer a public function,we should keep the inline docs as it makes it easier to understand what the fn is supposed to do along with the parameters

Reply:
Updated.